cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-security-administration-database-access, ACCESS>>                            | Database              | Determines whether a user can access a specific database.
+| <<administration-security-administration-database-startstop, START>>                          | Database              | Determines whether a user can start up a specific database.
+| <<administration-security-administration-database-startstop, STOP>>                           | Database              | Determines whether a user can stop a specific running database.
+| <<administration-security-administration-database-indexes, CREATE INDEX>>                     | Schema                | Determines whether a user is allowed to create indexes on a specific database.
+| <<administration-security-administration-database-indexes, DROP INDEX>>                       | Schema                | Determines whether a user is allowed to drop indexes on a specific database.
+| <<administration-security-administration-database-indexes, INDEX MANAGEMENT>>                 | Schema                | Determines whether a user is allowed to create and drop indexes on a specific database.
+| <<administration-security-administration-database-constraints, CREATE CONSTRAINT>>            | Schema                | Determines whether a user is allowed to create constraints on a specific database.
+| <<administration-security-administration-database-constraints, DROP CONSTRAINT>>              | Schema                | Determines whether a user is allowed to drop constraints on a specific database.
+| <<administration-security-administration-database-constraints, CONSTRAINT MANAGEMENT>>        | Schema                | Determines whether a user is allowed to create and drop constraints on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW NODE LABEL>>             | Schema                | Determines whether a user is allowed to create new node labels on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW RELATIONSHIP TYPE>>      | Schema                | Determines whether a user is allowed to create new relationship types on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>>          | Schema                | Determines whether a user is allowed to create new property names on a specific database.
+| <<administration-security-administration-database-tokens, NAME MANAGEMENT>>                   | Schema                | Determines whether a user is allowed to create new labels, types and property names on a specific database.
+| <<administration-security-administration-database-all, ALL DATABASE PRIVILEGES>>              | Database and schema   | Implies all privileges from the categories Database and Schema for a specific database.
```

Comment:
Database, Schema and Token

Reply:
Changed in text but not in `Category` column

Reply:
See other comment. Lets not call it Token. Let that be part of Schema